lib/OrdinaryDiffEqSSPRK/src/algorithms.jl

```diff
@@ -347,7 +347,7 @@ function KYKSSPRK42(stage_limiter!, step_limiter! = trivial_limiter!)
         False())
 end
 
-@doc explicit_rk_docstring("TBD", "SHLDDRK52")
+@doc explicit_rk_docstring("To be done", "SHLDDRK52")
```

Comment:
@ChrisRackauckas These above 3 solvers also need more info.

Reply:
The LDDRK methods are from https://www.sciencedirect.com/science/article/abs/pii/S0021999198959861

> The explicit Runge-Kutta (RK)
methods are widely used to discretize the time derivative because of their advantages that
include flexibility, large stability limits, and ease of programming. Hu and co-workers [2]
showed that the dissipation and dispersion properties of the RK methods depend on their
coefficients and optimized them for the convective wave equation, obtaining what they
called low-dissipation and dispersion Runge-Kutta (LDDRK) methods. These methods are
more efficient than classical ones, in terms of work required for a given accuracy, for wave
propagation problems.